packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewManager.kt

```diff
@@ -55,6 +55,11 @@ class ExpoImageViewManager(applicationContext: ReactApplicationContext) : Simple
     view.setResizeMode(resizeMode)
   }
 
+  @ReactProp(name = "fadeDuration")
+  fun setFadeDuration(view: ExpoImageView, fadeDuration: Int) {
+    view.setFadeDuration(fadeDuration)
```

Comment:
Let's strive for such syntax, shall we?
```suggestion
    view.fadeDuration = fadeDuration
```